.github/workflows/publish.yml

```diff
@@ -21,3 +21,28 @@ jobs:
       uses: JS-DevTools/npm-publish@v1
       with:
         token: ${{ secrets.NPM_TOKEN }}
+  github:
+    name: GitHub release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Get release version
+      id: get_version
+      run: |
+        export PACKAGE_VERSION=$(cat package.json | grep 'version' | sed 's/[ \",:]//g' | sed 's/version//')
+        echo "::set-output name=version::$PACKAGE_VERSION"
+    - name: Create and push git tag
+      uses: actions-ecosystem/action-push-tag@v1
+      with:
+        tag: ${{ steps.get_version.outputs.version }}
+        message: ${{ steps.commit.outputs.git-message }}
+    - name: Create release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ steps.get_version.outputs.version }}
+        release_name: Release ${{ steps.get_version.outputs.version }}
```

Comment:
I would be fine with this, but it may cause some confusion with the main repo as it doesn't use the `v` prefix. I agree that it looks nicer, but with 69 releases without the `v` switching seems a bit weird now...

This won't be a problem in general, I think, only if people try to checkout a specific version of the project in git, which is not often I'd guess. It might become really annoying if we ever decide to release this to [packagist](https://packagist.org/) as well, as it uses the tag value. In that case simple-icons would be installed using `x.y.z` and simple-icons-font using `vx.y.z`...

We definitely can make the switch to using the `v` prefix but I think we should discuss this with more maintainers + change it only with the next major release of simple-icons.

Reply:
I see :thinking:

Reply:
Can you bring this discussion to the attention of the other maintainers outside of this project?

Reply:
I'm not convinced now, if it has been used without "v" in the main repository so many releases, better leave it like that.